basic.test.js

```diff
@@ -0,0 +1,73 @@
+const durations = require('../src/durations.js');
+const MelodyQueue = require('../src/MelodyQueue.js');
+const waveGenerator = require('../src/WaveGenerator.js');
+const MelodyGenerator = require('../src/MelodyGenerator.js');
+const note = require('../src/note.js');
+
+// https://jestjs.io/docs/en/using-matchers
+// https://jestjs.io/docs/en/es6-class-mocks
+
+jest.mock('../src/MelodyQueue.js');
+
+beforeEach(() => {
+    MelodyQueue.mockClear();
+});
+
+// test('getNotesArray(["A4"]) should returns note("A4")', () => {
```

Comment:
commented code is a useless code!

Reply:
Agreed! I'll be sure to get rid of it in the future. :-D